### ISSUE TITLE
Prim path expression /World_0/franka/panda_rightfinger is invalid

### ISSUE THREAD

**Issue by SanMumumu** [arnold-benchmark/arnold#23]
Hi,

Thank you for share such a great work !

But I meet this error, can you tell me how to deal with it?

`Exception: Prim path expression /World_0/franka/panda_rightfinger is invalid, a prim matching the expression needs to created before wrapping it as view`

**Comment by nikepupu** [comment]
It seems that the robot is not loaded. Can you make sure that the assets are placed in the correct location?

**Comment by SanMumumu** [comment]
Hi, 

Thanks for your reply. You are right. 

And I want to know how much GPU memory is needed to complete all the eval code ?  The memory is out on the RTX3060 already.”

Do you have any suggestions for saving GPU memory? Is swapping the GPU the only option?

**Comment by nikepupu** [comment]
Are you running with headless mode?

**Comment by SanMumumu** [comment]
Yes, in default.py: `state_head: 0`

The weight file is saved after training. And we changed the code to distributed training to save traning memory.

But when I run ckpt_selection.py, it will be OOM.

**Comment by nikepupu** [comment]
Unfortunately the code is only tested with 24G VRAM GPUs, maybe you can change the model to bfloat16 for training? You can reduce the number of cameras to save VRAMs too.

**Comment by SanMumumu** [comment]
Ok, I konw, thank you.

**Comment by SanMumumu** [comment]
I'm sorry, but I want to ask an unrelated question. 

I used a remote desktop connection to connect to the server.

**1.**  I initially installed the NVIDIA Omniverse platform on the server and then installed `IsaacSim 2022.1.1`. But it will show `segment error`.
**2.**  So I installed the latest one IsaacSim `2023.1.1`. But clicking Launch under LIBRARY doesn't respond. 

However, when I use the mainframe, the above problem will not appear. This is so strange, 
Is it best not to install these virtual environments or Isaac sim on the server?
